site/en/setup.md

```diff
@@ -14,48 +14,57 @@ to achieve the workflow best suited for OCRing your content.
 ## System requirements
 
 Minimum system requirements
-* 8 GB RAM (more recommended)
-  * The more RAM is available, the more concurrent processes can be run
-  * Exceedingly large images (newspapers, folio-size books...) require a lot of RAM
-* min. 100 GB free disk space for local installation 
-  * ~10 GB for [`ocrd_all`](https://github.com/OCR-D/ocrd_all), including dependencies
-  * ~10 GB for various [pretrained models](https://ocr-d.de/en/models)
-  * ~10 GB for training and evaluation data for training, as well as trained (intermediate) models
-  * ~20 GB for data to process
-  * double the space requirements to be able to [isolate virtualenvs](https://github.com/OCR-D/ocrd_all/pull/118) and store backups of (intermediate) data
-* Operating system: Ubuntu 18.04
-  * Ubuntu 18.04 is our target platform because it was the most up-to-date Ubuntu LTS release when we started developing and [will be supported for the foreseeable future](https://ubuntu.com/about/release-cycle)
-  * Other Linux distributions or Ubuntu versions can also be used, though some instructions have to be adapted (e.g. package management, locations of some files)
-  * With Windows Subsystem for Linux (WSL), a feature of Windows 10, it is [also possible to set up an Ubuntu 18.04 installation within Microsoft Windows](https://github.com/OCR-D/ocrd-website/wiki/OCR-D-on-Windows)
-  * OCR-D can be deployed on an [Apple MacOSX machine using Homebrew](https://github.com/OCR-D/ocrd-website/wiki/OCR-D-on-macOS)
-* Python 3.5, 3.6 or 3.7
-  * OCR-D's target Python version is currently Python 3.5 which we will continue to support until at least the end of 2020
-  * Python 3.6 and 3.7 are also tested and supported
-  * We currently **cannot fully support Python 3.8**, because there currently (July 2020) are no pre-built Python packages for Tensorflow and other GPU related software). We expect to unconditionally support Python 3.8 in Q4 2020 or Q1 2021 the latest.
-
+<details>
+<summary>- 8 GB RAM (more recommended)</summary>
+  
+  - The more RAM is available, the more concurrent processes can be run
+  - Exceedingly large images (newspapers, folio-size books...) require a lot of RAM
+  
+</details>
+<details>
+<summary>- min. 100 GB free disk space for local installation</summary>
```

Comment:
Maybe this would also be a good moment to reduce the HDD storage requirements. The idea was to prevent users from provisioning too little storage space, but 100 GB is quite a daunting number. IIRC that was also an issue raised by @cneud.

Reply:
IMHO it could be made more clear that a minimum of ~7-10 GB does indeed suffice for installation of `ocrd_all` (pending models) and that more disk space may be very useful but really required only depending on the actual usage.

Also these other values seem quite arbitrary and will depend only on the user:
- "10 GB for evaluation data"?
- "20 GB for data to process"?

Finally, when would I benefit from isolated venvs? Is this mandatory, optional, in which cases? As end user I cannot tell.

Reply:
I concur we should be more realistic about the actual minimum required disk space, and omit figures we cannot really estimate.

> Finally, when would I benefit from isolated venvs? Is this mandatory, optional, in which cases? As end user I cannot tell.

There's no alternative to isolated venvs (besides isolated Docker containers) if you want to install multiple OCR-D modules, because of the Python dependency hell (and `pip` shortcomings). So for ocrd_all it's a mandatory solution, for native installation you're on your own (but _will_ suffer the effects). The toll of this scheme on disk space requirements is considerable because Tensorflow and its dependencies require GBs each time. The setup guide does not even mention this (implementation detail), but the linked ocrd_all README does discuss it.

Reply:
Thanks for elaborating, that's what I assumed. So I take it this is mandatory then and perhaps the according sentence may even be omitted from this guide as it now presents this as a user choice. 

Are the ~7GB minimum already including this?

Reply:
> Thanks for elaborating, that's what I assumed. So I take it this is mandatory then and perhaps the according sentence may even be omitted from this guide as it now presents this as a user choice.

Indeed, that's what it sounds like. Also I don't quite understand what backups are required in this context (since everything is repo-controlled already or built by make), so perhaps that line should be omitted entirely?

> Are the ~7GB minimum already including this?

With the standard Docker installation (`ocrd/all:maximum-git` i.e. all except `DISABLED_MODULES ?= cor-asv-fst opencv-python ocrd_kraken clstm ocrd_ocropy`) we are already at 15.4 GB. (I have no idea how Dockerhub counts...)

BTW I am surprised to see that the non-git Docker images are still smaller than the git ones, as they are merely aliases. Does anyone have a clue what's going on?